### PR TITLE
Create nested mappings for sub nested schema

### PR DESCRIPTION
**Pull request by malathi13**

### CODE REVIEW

File: bigquery/indexer.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import logging
 import os
 import time
-
+import pdb
```
Comment (bfcrampton) [inline bigquery/indexer.py]: nit: rm

File: bigquery/indexer.py [hunk 2]
```diff
@@ -98,6 +101,8 @@ def _create_nested_mappings(es, index_name, table, sample_id_column):
     # queries will work correctly, see:
     # https://www.elastic.co/guide/en/elasticsearch/reference/6.4/nested.html#_how_arrays_of_objects_are_flattened
     nested = _get_nested_mappings(table.schema, _table_name_from_table(table))
+    print("Final nested")
```
Comment (bfcrampton) [inline bigquery/indexer.py]: nit: rm?

File: bigquery/indexer.py (hunk 1 above)
Comment (malathi13) [reply bigquery/indexer.py]: Done

File: bigquery/indexer.py (hunk 2 above)
Comment (malathi13) [reply bigquery/indexer.py]: Done

**Pull request merged**